### DISCUSSION TITLE
Is it possible to organize imports on format?

### DISCUSSION THREAD

**Discussion by tomtitherington** [microsoft/vscode-isort#385] [Q&A] [answered]
Similar to how you can organize imports on save via:

```
"[python]": {
    "editor.codeActionsOnSave": {
      "source.organizeImports": "explicit"
    }
  }
```

It would be nice to organize these imports once the formatter has run. Is this possible?

**Comment by karthiknadig** [comment, marked as answer]
@tomtitherington This is something we can't control from the isort extension directly, as this is a behavior controlled by the editor. You would need to file a feature request on VS Code.

The only option right now is to trigger it manually after formatting.

**Reply by tomtitherington** [reply]
Ah okay, thanks for clarifying @karthiknadig
